src/core/render/emojify.js

```diff
@@ -35,6 +35,8 @@ export function emojify(text, useNativeEmoji) {
       )
       // Mark colons in comments
       .replace(/<!--[\s\S]+?-->/g, m => m.replace(/:/g, '__colon__'))
+      // Mark colons in URIs
+      .replace(/[a-z]{2,}:\/\/[^\s]+/gi, m => m.replace(/:/g, '__colon__'))
```

Comment:
This regex will miss URLs that omit the protocol:

```html
<a href="//domain.com/:smile/">Smile</a>
```

Reply:
I was trying to be a strict as possible with the regex to reduce the number of false positives. Happy to update the expression to make `scheme:` optional.

The root cause of my issue was related to absolute SharePoint urls being pasted into markdown which were all absolute and included the scheme.

Reply:
Note that //domain.com/:smile:/ doesn't https://domain.com/:smile:/ appear to be automatically parsed as a URI like a fully qualified URI does by the markdown parser.

What do you prefer? Optional versus required `scheme:`?

Reply:
Fixed